src/test/lockDistribution.t.sol

```diff
@@ -148,17 +150,20 @@ contract LockDistributionTest is DSTest {
         *  there after the error in assert_not_contract() is present for all activity with vote escrow fucntions associated to the address eoa1
         */
         distribution.lockDistribution(block.timestamp + 94608000);  //distr calls vote escrow | 4yr = 126144000 | 3 yr = 94608000
-        //cheats.warp(block.timestamp + 3 days);
-        //voteEscrow.increase_unlock_time(block.timestamp + 110000000);
+        cheats.warp(block.timestamp + 3 days);
+        voteEscrow.increase_unlock_time(block.timestamp + 110000000);
 
         emit log_named_uint("ve balance for eoa1", voteEscrow.balanceOf(eoa1));
 
-        cheats.warp(block.timestamp + 94608001);
+        cheats.warp(block.timestamp + 110000001);
         emit log_named_uint("ve balance for eoa1", voteEscrow.balanceOf(eoa1));
 
         voteEscrow.withdraw();
 
-        voteEscrow.create_lock(amount1, block.timestamp + 21 days);
+        baoToken.approve(address(voteEscrow), baoToken.balanceOf(eoa1));
```

Comment:
@N0xMare this function was missing and causing reverts after I solved the whitelist problem